prometheus/base/prometheus.rules.yaml

```diff
@@ -11,3 +11,9 @@ spec:
       rules:
         - record: o11y_prometheus_http_requests_total
           expr: sum by (code) (prometheus_http_requests_total)
+        - record: container:network_transmit_bytes:group
```

Comment:
The format of this metric name is inconsistent with the example above. We should establish a naming convention according to [best practices](https://prometheus.io/docs/practices/naming/#metric-names). It seems that we should use `appstudio` ([thread](https://redhat-internal.slack.com/archives/C02CTEB3MMF/p1678308775308909)) for the application prefix.

I suggest we remove the example above and use `appstudio_container_network_transmit_bytes_total` here. Wdyt?

Reply:
I don't mind. This is just something I had lying around when creating it. I'll change it.

Reply:
done